contrib/helm/harbor/templates/registry/registry-cm.yaml

```diff
@@ -17,8 +17,10 @@ data:
       {{- end }}
       cache:
         layerinfo: inmemory
+      {{- if not .Values.registry.objectStorage }}
       filesystem:
         rootdirectory: /var/lib/registry
+      {{- end }}
```

Comment:
we can move this up in the 1st {{- if .Values.registry.objectStorage }} by adding a 'else' there.